build/Targets/Versions.props

```diff
@@ -49,6 +49,8 @@
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' == 'Windows_NT'">$(UserProfile)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' != 'Windows_NT'">$(HOME)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+
+    <Version Condition="$(Version) == ''">$(NuGetPerBuildPreReleaseVersion)</Version>
```

Comment:
Ah, I get a repro using netcoreapp1.1 and v1.0 SDK. runtimeconfig.json comes out empty. Please do file the bug.